helm_deploy/values-prod.yaml

```diff
@@ -33,5 +33,8 @@ generic-prometheus-alerts:
   alertSeverity: digital-prison-service
   rdsAlertsDatabases:
     cloud-platform-5b6e8eb8ba215b7b: "Manage Adjudications API"
+  sqsAlertsQueueNames:
```

Comment:
I checked the output of https://github.com/ministryofjustice/cloud-platform-environments/pull/13271 and the queue name is prefixed by `dps-core` rather than `"hmpps-adjudications` so also updated https://github.com/ministryofjustice/hmpps-manage-adjudications-api/pull/385/files#diff-4394770745edad3a353d7e224cd427beb723c3dcb68dc356a319b609acd1f836L23-L24